alts/src/main/java/io/grpc/alts/internal/AltsTsiHandshaker.java

```diff
@@ -178,6 +182,17 @@ public TsiFrameProtector createFrameProtector(int maxFrameSize, ByteBufAllocator
     byte[] key = handshaker.getKey();
     Preconditions.checkState(key.length == AltsChannelCrypter.getKeyLength(), "Bad key length.");
 
+    // In case the peer does not send max frame size (e.g. peer is gRPC Go or peer uses an old
+    // binary), the negotiated frame size is set to MIN_FRAME_SIZE. Otherwise, the negotiated
+    // value is based on peer frame size. maxFrameSize parameter is ignored.
+    maxFrameSize = AltsHandshakerClient.getMinFrameSize();
```

Comment:
As in my other comment above, don't set maxFrameSize here. 
If there is no frame size negotiation, use the value from parameter -- which will be the default frame size